packages/http/src/mocker/HttpMocker.ts

```diff
@@ -63,7 +63,7 @@ Here is the original validation result instead: ${JSON.stringify(input.validatio
     const example = negotiationResult.example as INodeExample;
     if (example && example.value !== undefined) {
       body = example.value;
-    } else {
+    } else if (negotiationResult.schema) {
```

Comment:
Would be nice to cover this with a unit test.

Reply:
Good point; I'll check whether it makes sense and add it!

Reply:
Did it make sense? :P

Reply:
Actually this is covered [here](https://github.com/stoplightio/prism/blob/develop/packages/http/src/mocker/__tests__/HttpMocker.spec.ts#L95-L139) and does not require a dedicated test. All 3 branches are tested and the test is exactly checking out that.